sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -74,11 +72,6 @@ class CustomerIO internal constructor(
             return this
```

Comment:
👏 Thanks, Android lint 👏. Without the lint warning here, I wouldn't have noticed this. 

This line of code is a bug. Should be:
```kt
this.shouldAutoRecordScreenViews = shouldTrackDeviceAttributes
```

Phew. Wouldn't have noticed that if it wasn't for the lint. 

Can we add to our `Builder` unit tests to test that this property gets set properly?